# HG changeset patch
# User Michael Godfrey <michaeldgodfrey@gmail.com>
# Date 1384099618 18000
#      Sun Nov 10 11:06:58 2013 -0500
# Node ID e5566719e0a16532a8179bab6b0e4e7964faa9f2
# Parent  6e43fbab4b29cef398e893be01845dd533c5d732
doc: Mention GUI in introduction.  Update figure root properties.

* intro.txi: Mention GUI in introduction.

* plot.txi: Update figure root properties.

diff --git a/doc/interpreter/intro.txi b/doc/interpreter/intro.txi
--- a/doc/interpreter/intro.txi
+++ b/doc/interpreter/intro.txi
@@ -16,20 +16,25 @@
 @c along with Octave; see the file COPYING.  If not, see
 @c <http://www.gnu.org/licenses/>.
 
 @node Introduction
 @chapter A Brief Introduction to Octave
 @cindex introduction
 
 GNU Octave is a high-level language, primarily intended for numerical
-computations.  It provides a convenient interactive command line 
-interface for solving linear and nonlinear problems numerically, and 
-for performing other numerical experiments.  It may also be used as a 
-batch-oriented language for data processing.
+computations.  It is typically used for such problems as solving 
+linear and nonlinear equations, numerical linear algebra, statistical
+analysis, and for performing other numerical experiments.  It may also
+be used as a batch-oriented language for automated data processing.
+
+Until recently GNU Octave provided a command-line interface with
+graphical results displayed in separate windows.  The current version
+(version 3.8, released in late 2013) also provides, by default, a
+graphical user interface.
 
 GNU Octave is freely redistributable software.  You may redistribute 
 it and/or modify it under the terms of the GNU General Public License
 as published by the Free Software Foundation.  The GPL is included in 
 this manual, @pxref{Copying}.
 
 This manual provides comprehensive documentation on how to install, 
 run, use, and extend GNU Octave.  Additional chapters describe how 
@@ -41,20 +46,23 @@ This document corresponds to Octave vers
 * Running Octave::
 * Simple Examples::
 * Conventions::
 @end menu
 
 @node Running Octave
 @section Running Octave
 
-On most systems, Octave is started with the shell command 
-@samp{octave}.  Octave displays an initial message and then a prompt
-indicating it is ready to accept input.  You can begin typing Octave
-commands immediately afterward.
+On most systems, Octave is started with the shell command @samp{octave}.
+This, by default, starts the graphical user interface (GUI).  The central
+window in the GUI is the Octave command-line interface.  Octave displays
+an initial message and then a prompt indicating it is ready to accept
+input.  If you have chosen the traditional command-line interface only
+the command prompt appears.  In any case, you can immediately begin
+typing Octave commands. 
 
 If you get into trouble, you can usually interrupt Octave by typing
 @kbd{Control-C} (written @kbd{C-c} for short).  @kbd{C-c} gets
 its name from the fact that you type it by holding down @key{CTRL} and
 then pressing @key{c}.  Doing this will normally return you to Octave's
 prompt.
 
 @cindex exiting octave
diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -1039,40 +1039,41 @@ Thus, for example:
 
 @smallexample
 h = figure ();
 get (h, "type")
 ans = figure
 get (h, "");
 error: get: ambiguous figure property name ; possible matches:
 
-__graphics_toolkit__  hittest              resize
-__enhanced__          integerhandle        resizefcn
-__modified__          interruptible        selected
-__myhandle__          inverthardcopy       selectionhighlight
-__plot_stream__       keypressfcn          selectiontype
-alphamap              keyreleasefcn        tag
-beingdeleted          menubar              toolbar
-busyaction            mincolormap          type
-buttondownfcn         name                 uicontextmenu
-children              nextplot             units
-clipping              numbertitle          userdata
-closerequestfcn       paperorientation     visible
-color                 paperposition        windowbuttondownfcn
-colormap              paperpositionmode    windowbuttonmotionfcn
-createfcn             papersize            windowbuttonupfcn
-currentaxes           papertype            windowscrollwheelfcn
-currentcharacter      paperunits           windowstyle
-currentobject         parent               wvisual
-currentpoint          pointer              wvisualmode
-deletefcn             pointershapecdata    xdisplay
-dockcontrols          pointershapehotspot  xvisual
-doublebuffer          position             xvisualmode
-filename              renderer
-handlevisibility      renderermode
+__enhanced__           hittest                resize
+__graphics_toolkit__   integerhandle          resizefcn
+__guidata__            interruptible          selected
+__modified__           inverthardcopy         selectionhighlight
+__myhandle__           keypressfcn            selectiontype
+__plot_stream__        keyreleasefcn          tag
+alphamap               menubar                toolbar
+beingdeleted           mincolormap            type
+busyaction             name                   uicontextmenu
+buttondownfcn          nextplot               units
+children               numbertitle            userdata
+clipping               outerposition          visible
+closerequestfcn        paperorientation       windowbuttondownfcn
+color                  paperposition          windowbuttonmotionfcn
+colormap               paperpositionmode      windowbuttonupfcn
+createfcn              papersize              windowkeypressfcn
+currentaxes            papertype              windowkeyreleasefcn
+currentcharacter       paperunits             windowscrollwheelfcn
+currentobject          parent                 windowstyle
+currentpoint           pointer                wvisual
+deletefcn              pointershapecdata      wvisualmode
+dockcontrols           pointershapehotspot    xdisplay
+doublebuffer           position               xvisual
+filename               renderer               xvisualmode
+handlevisibility       renderermode
 @end smallexample
 
 The root figure has index 0.  Its properties may be displayed by:
 @code{get (0, "")}.
 
 The uses of @code{get} and @code{set} are further explained in
 @ref{XREFget,,get}, @ref{XREFset,,set}.
 
